sympy/geometry/point.py

```diff
@@ -738,21 +825,19 @@ def is_concyclic(*points):
             return False
         if len(points) <= 2:
             return True
-        points = [Point.pointify(p) for p in points]
+        points = Point.normalize_dimensions(2, *[
+            Point.pointify(p) for p in points])
         if len(points) == 3:
             return (not Point.is_collinear(*points))
 
-        try:
-            from .ellipse import Circle
-            c = Circle(points[0], points[1], points[2])
-            for point in points[3:]:
-                if point not in c:
-                    return False
-            return True
-        except GeometryError:
-            # Circle could not be created, because of collinearity of the
-            # three points passed in, hence they are not concyclic.
+        if Point.is_collinear(*points[:3]):
             return False
+        from .ellipse import Circle
+        c = Circle(*points[:3])
+        for point in points[3:]:
+            if point not in c:  # XXX does this raise error if it can't be determined?
```

Comment:
IIRC, `in` doesn't is binary but `contains` is trinary with `True` `False` `None`